.github/workflows/ruby_test.yml

```diff
@@ -82,11 +82,15 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           bundler-cache: true # runs 'bundle install' and caches installed gems automatically
-      # Establish a cache of js modules to improve performance
+      - name: Read .nvmrc
+        run: echo ::set-output name=NVMRC::$(cat .nvmrc)
+        id: nvm
       - name: Setup node
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v1
```

Comment:
I didn't notice this last week but you have downgraded the setup-node version? I checked on some of other repos and we are using v4.

Reply:
I think I've tested this with v1. I made the change to read the node version from `.nvmrc` but forgot to upgrade the actions version. I'll upgrade it to `v4` and do a CI. Thanks @stevieing for pointing this out.